openapi-generator.sh

```diff
@@ -0,0 +1,27 @@
+# Before running this:
```

Comment:
Currently this only works locally. We would have to figure out how/when to run this in CI, and ideally maybe even publish the npm module rather than linking it locally?